content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than ``2.164+``footnote:[It was possible to run Jenkins on Java 11 since the version `2.155` but its usage was then more complex. We do not recommend to use any version prior to `2.164`.].
+
+For the sake of simplicity, we will describe next how to run Jenkins on Java 11 with the most recent version of Jenkins.
+
+NOTE: We do encourage you to only use LTS versions for production purposes. 
+At the time of this writing, there is no LTS version with Java 11 Support.
+
+== Running Jenkins with Docker
+
+The easiest way to run Jenkins on Java 11 is to use the Docker image.
+All these images are based on the official link:https://hub.docker.com/r/_/openjdk/[openjdk:11-jdk] image maintained by the Docker Community.
+
+You can choose these Java 11 based images by using the `jdk11` tag (for the latest weekly release) or by suffixing the tag to use with `-jdk11` (eg. `2.164-jdk11`).
+
+This is the command to use, in order to start a Jenkins running on Java 11: 
+
+[source, shell]
+----
+docker pull jenkins/jenkins:jdk11
+docker run --rm -ti \
+  -p 8080:8080 -p 50000:50000
+  -v jenkins-home:/var/jenkins_home \
+  jenkins/jenkins:jdk11
+----
+
+== Running Jenkins with Java
+
+It is also possible to run Jenkins using the `java` command.
+
+Starting Jenkins would be:
+
+[source, shell]
+----
+java -jar jenkins.war
```

Comment:
```suggestion
java -jar jenkins.war --httpPort=8080
```

To make the result similar to Docker

Reply:
By default, port when not specifiied is `8080`, no? I tried to make it over simplified to show it's easy. In the end, we should never start Jenkins this way manually.

Reply:
yes, it is right